### PR TITLE
flameshot: Use Qt mkDerivation

### PR DESCRIPTION
**Pull request by MasseR**
###### Things done

- [X] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

###### Notify maintainers

cc @scode

### CODE REVIEW

File: pkgs/tools/misc/flameshot/default.nix [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
+{ mkDerivation, lib, fetchFromGitHub, qtbase, qmake, qttools, qtsvg }:
 
 # To use `flameshot gui`, you will also need to put flameshot in `services.dbus.packages`
 # in configuration.nix so that the daemon gets launched properly:
 #
 #   services.dbus.packages = [ pkgs.flameshot ];
 #   environment.systemPackages = [ pkgs.flameshot ];
-stdenv.mkDerivation rec {
+mkDerivation rec {
   name = "flameshot-${version}";
```
Comment (teto) [inline pkgs/tools/misc/flameshot/default.nix]: you could use pname as well

Comment (MasseR) [reply pkgs/tools/misc/flameshot/default.nix]: How important is that change? I would like to avoid unrelated changes.

Comment (teto) [reply pkgs/tools/misc/flameshot/default.nix]: don't mind it, you could have replaced `name = "flameshot-${version}";` with `pname = "flameshot"`. `name` is then generated by mkDerivation

Comment (Mic92) [reply pkgs/tools/misc/flameshot/default.nix]: fixed.

**Pull request merged**